### PR TITLE
Exclude CONTRIBUTING file from autobuild

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**

Follow up from #2532 I missed CONTRIBUTING.md

CHANGELOG: SKIP

**Context: Fixes an issue?**

No. Related to https://github.com/jamulussoftware/jamulus/pull/2561#issuecomment-1081154176

**Does this change need documentation? What needs to be documented and how?**

No

**Status of this Pull Request**

Ready to merge

**What is missing until this pull request can be merged?**

Nothing

## Checklist

-  [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
-  [x] I tested my code and it does what I want
-  [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
-  [x] I've filled all the content above

### CODE REVIEW

File: .github/workflows/autobuild.yml [hunk 1]
```diff
@@ -43,6 +44,7 @@ on:
       - '**README.md'
       - 'docs/**'
       - 'SECURITY.md'
+      - 'CONTRIBUTING.md'
```
Comment (hoffie) [inline .github/workflows/autobuild.yml:47]: Different list order than the list above now... :)

**Review by hoffie — APPROVED** [review]

File: .github/workflows/autobuild.yml (hunk 1 above)
Comment (ann0see) [reply .github/workflows/autobuild.yml:47]: Thanks. Fixed

**Comment by ann0see** [comment]
Should be squash merged

**Comment by ann0see** [comment]
Any other main dev to review this?

**Comment by pljones** [comment]
I don't like what's happening with CodeQL on these PRs.  It's offputting.  Is something wrong with it?

**Comment by pljones** [comment]
OK, sorted, whatever it was.

**Review by pljones — APPROVED** [review]

**Pull request merged**